babel.config.js

```diff
@@ -53,6 +53,12 @@ module.exports = function(api) {
           useBuiltIns: true
         }
       ],
+      [
+        '@babel/plugin-proposal-private-property-in-object',
+        {
+          loose: true
```

Comment:
Is this for speed/performance?

Reply:
It was fixing a deprecation warning. Essentially, this is what it was doing anyway, thanks to a similar configuration option elsewhere. I had a bit of a look at what it actually means, and loose is essentially faster, with slightly better compatibility, as the cost of not being quite so strict. I decided to maintain existing behaviour.